pennylane/ops/op_math/controlled.py

```diff
@@ -837,7 +837,7 @@ def name(self):
 
     @property
     def grad_method(self):
-        return self.base.grad_method
+        return "A" if isinstance(self.base, qml.GlobalPhase) else self.base.grad_method
```

Comment:
Rather than doing this special handling for `GlobalPhase` should we do this whenever `base.grad_method` is None.
```python
     return "A" if self.base.grad_method is None else self.base.grad_method
```

Reply:
Could keep this the same even if `GlobalPhase.grad_method = "F"`.